### PR TITLE
feat(vault): update module

### PR DESCRIPTION
**Pull request by leylmordor**
This PR will:
- change health default value
- add support for labels

### CODE REVIEW

File: modules/cluster/variables.tf [hunk 1]
```diff
@@ -331,7 +331,7 @@ variable "min_ready_sec" {
 variable "hc_initial_delay_secs" {
   description = "The number of seconds that the managed instance group waits before it applies autohealing policies to new instances or recently recreated instances."
   type        = number
-  default     = 60
+  default     = 300
```
Comment (zacaaron) [inline modules/cluster/variables.tf:334]: @leylmordor why the change from 1 min to 5 mins? That's quite a long wait

Comment (leylmordor) [reply modules/cluster/variables.tf:334]: Yes, this was done when we last changed rhe VM type. This is to address the issue where vault instances gets replaced from time to time.

Comment (zacaaron) [reply modules/cluster/variables.tf:334]: OK got it

**Review by zacaaron — APPROVED** [review]
lgtm
